cluster/service.go

```diff
@@ -127,9 +127,9 @@ func (s *Service) handleConn(conn net.Conn) {
 		conn.Close()
 	}()
 
-	s.Logger.Printf("accept remote write connection from %v\n", conn.RemoteAddr())
```

Comment:
Both read and write connections go through this service, since remote-mapping was implemented. However this log message was never updated.